### PR TITLE
added functionality to open authorization url in browser

### PR DESCRIPTION
**Pull request by miyuchina**
Hi,

I was wondering if you are interested in adding a feature that automatically opens the QR code in user's default browser, so that they don't need to copy/paste in the URL.

Thanks for the package.

### CODE REVIEW

**Comment by justdoit0823** [comment]
Cool, it's useful for users write a terminal program with this package. @miyuchina Can you edit those examples with the new function as well?

**Comment by miyuchina** [comment]
Should be working now!

File: pywxclient/core/client.py [hunk 1]
```diff
@@ -124,6 +124,13 @@ def get_authorize_url(self):
         self._uuid = uuid
         return self._api_cls.get_qrcode_url(self.session, uuid)
 
+    def open_authorize_url(self):
+        """Open WeChat authorization url in system-default browser."""
+        import webbrowser       # stdlib
```
Comment (justdoit0823) [inline pywxclient/core/client.py]: It's more standardized to import system standard module in the file header as described at [pep8-imports](https://www.python.org/dev/peps/pep-0008/#imports). And following this rule, it's more friendly to other contributors.

Comment (miyuchina) [reply pywxclient/core/client.py]: Ah, I was mainly considering that you didn't add this function in the first place, and putting imports inside the function defers the memory penalty until someone actually uses `open_authorize_url` itself. Fixed now!

**Comment by justdoit0823** [comment]
@miyuchina Thanks. What you have done will make it easier and convenient for users.

**Pull request merged**